app/assets/stylesheets/application-webpack.scss

```diff
@@ -0,0 +1,6 @@
+$pf-global--enable-reset: false;
+$pf-global--disable-fontawesome: true;
+
+@import '~@fortawesome/fontawesome-free/css/all.css';
```

Comment:
fortawesome != fontawesome?

Reply:
Apparently OK, https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers